source/class/qx/tool/cli/commands/package/Migrate.js

```diff
@@ -180,6 +180,13 @@ qx.Class.define("qx.tool.cli.commands.package.Migrate", {
           }
         }
       }
+      let compileJsFilename = path.join(process.cwd(), "compile.js");
+      if (await fs.existsAsync(compileJsFilename)) {
+        let data = await fs.readFileAsync(compileJsFilename, "utf8");
+        if (data.indexOf("module.exports") < 0) {
+          console.warn("*** Your compile.js appears to be missing a `module.exports` statement - please see https://git.io/fjBqU for more details");
```

Comment:
You can add `needFix = true;` here then it will not proceed until the user fixes the issue (which s/he should).

Reply:
wait, no, that's not how the code works.

Reply:
I'll adapt it later...